migrations/Version20180207095757.php

```diff
@@ -31,8 +31,8 @@ public function up(Schema $schema): void
         $this->addSql('CREATE INDEX IDX_C395A618615D27E1 ON Job (backupLocation_id)');
         
         //get backup_dir param and store as new BackupLocation
-        if ($this->backupDir != null) {
-            $location = $this->container->getParameter('backup_dir');
+        if ($this->backup_dir != null) {
+            $location = $this->backup_dir;
             $this->addSql("INSERT INTO BackupLocation VALUES(1,'Default','','" . $location . "',0)");
```

Comment:
Let's call things properly; this isn't an inconsistency fix; trait's member value is backup_dir so we are fixing a bug here, because otherwhise we're checking a non-existent attribute. (this would always fallback to else clause I guess)